__tests__/app.test.js

```diff
@@ -115,6 +115,27 @@ describe("/api/articles", () => {
             }
         }
 
+        test("200: ?topic=cats \tResponds with articles array filtered by topic.", () => {
+            return request(app)
+            .get("/api/articles?topic=cats")
+            .expect(200)
+            .then(( { body: { articles }}) => {
+                expect(articles).toHaveLength(1)
+                articles.forEach( article => {
+                    expect(article.topic).toBe("cats")
+                })
+            })
+        })
+
+        test("200: ?topic=lizards \tResponds with an empty articles array for topic values not in the database.", () => {
```

Comment:
do we want an empty array when the topic doesn't exist ? what if the topic exists but there are no articles under that topic.